src/Microsoft.CodeAnalysis.Analyzers/Core/CodeAnalysisDiagnosticsResources.resx

```diff
@@ -586,4 +586,13 @@
   <data name="SemanticModelGetDeclaredSymbolAlwaysReturnsNullForFieldMessage" xml:space="preserve">
     <value>A call to 'SemanticModel.GetDeclaredSymbol({0})' will always return 'null'</value>
   </data>
+  <data name="DoNotDeclareCompilerFeatureInAssemblyWithUnsupportedTargetFrameworkMessage" xml:space="preserve">
```

Comment:
>DoNotDeclareCompilerFeatureInAssemblyWithUnsupportedTargetFramework [](http://example.com/codeflow?start=14&length=67)

make the names consistent, i.e. `DoNotDeclareCompilerFeatureInAssemblyWithUnsupportedTargetFramework vs `DoNotRegisterCompilerTypesWithBadTargetFramework`

Reply:
➡️ Updated